igor-monitor-travis/src/main/java/com/netflix/spinnaker/igor/travis/TravisCache.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2018 Netflix, Inc.
+ * Copyright 2020 Schibsted ASA.
```

Comment:
Mind the copyright changes that alter companies.

Reply:
Oh, sorry. I did change a whole lot of them back, but seems this one slipped though the cracks. Bad IntelliJ!
I’ll fix it in the morning.